internal/controller/pipelinerollout_controller.go

```diff
@@ -370,8 +370,8 @@ func (r *PipelineRolloutReconciler) reconcile(
 	// Object already exists
 	// if Pipeline is not owned by Rollout, fail and return
 	if !checkOwnerRef(existingPipelineDef.OwnerReferences, pipelineRollout.UID) {
-		pipelineRollout.Status.MarkFailed(fmt.Sprintf("Pipeline %s already exists in namespace", pipelineRollout.Name))
-		return false, nil
+		numaLogger.Debugf("PipelineRollout %s failed because Pipeline %s already exists in namespace", pipelineRollout.Name, existingPipelineDef.Name)
+		return false, fmt.Errorf("pipeline %s already exists in namespace", existingPipelineDef.Name)
```

Comment:
```suggestion
		errStr := fmt.Sprintf("Pipeline %s already exists in namespace, not owned by PipelineRollout", existingPipelineDef.Name)
		numaLogger.Debugf("PipelineRollout %s failed because %s", pipelineRollout.Name, errStr)
		return false, fmt.Error(errStr)
```